async_processor/types.py

```diff
@@ -288,6 +302,21 @@ def to_output(self) -> Output:
         return SQSOutput(self)
 
 
+class AMQPOutputConfig(OutputConfig):
+    type: constr(regex=r"^amqp$") = "amqp"
+
+    url: constr(
+        regex=r"^(?:amqp|amqps):\/\/(?:([^:/?#\s]+)(?::([^@/?#\s]+))?@)?([^/?#\s]+)(?::(\d+))?\/?([^?#\s]*)?(?:\?(.*))?$"
+    )
+    queue_name: str
```

Comment:
Ideally this should be

```suggestion
    exchange_name: str
    routing_key: str
```